### DISCUSSION TITLE
运行frpc时报错

### DISCUSSION THREAD

**Discussion by Ganlin-Lu** [natfrp/wiki#76] [使用问题] [answered]
环境 CentOS 8 Stream，root用户
运行程序的命令./frpc -v 报错：段错误 (核心已转储)。这是为什么呢？
![image](https://user-images.githubusercontent.com/49240669/172276369-4e16abf4-7a48-4b86-b6c9-f012ab7a1463.png)

**Comment by Ganlin-Lu** [comment]
眼瞎了，是CentOS 9 Stream.

**Comment by fakeboboliu** [comment, marked as answer]
你好，CentOS Stream 9 有一些神奇的安全功能会与我们使用的 upx 冲突，请在下载 url 后面加上 `_noupx` 下载未经压缩的版本。

我们预期会在下个版本处理这些问题。

又：右上角三个点中可以编辑
